### ISSUE TITLE
[C#] Implement new Concept Exercise: conditionals-ternary

### ISSUE THREAD

**Issue by mikedamay** [exercism/v3#1635]
This issue describes how to implement the `conditionals-ternary` concept exercise for the C# track.

## Getting started

**Please please please read the docs before starting.** Posting PRs without reading these docs will be a lot more frustrating for you during the review cycle, and exhaust Exercism's maintainers' time. So, before diving into the implementation, please read up on the following documents:

- [The features of v3](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md).
- [Rationale for v3](https://github.com/exercism/v3/blob/master/docs/rationale-for-v3.md).
- [What are concept exercise and how they are structured?](https://github.com/exercism/v3/blob/master/docs/features-of-v3.md)

Please also watch the following video:

- [The Anatomy of a Concept Exercise](https://www.youtube.com/watch?v=gkbBqd7hPrA).

## Goal

The goal of this exercise is to teach the student how and when to use ternary expressions  and the `?:` operator in C#.

## Learning objectives

- Know how to use ternary expressions - using the `?:` operator.
- Know how to use `throw` expressions.

## Out of scope

Nothing notable

## Concepts

- ternary expressions
- `throw` expressions
- `exceptions`

## Prerequisites

- `booleans`
- `conditionals-if`

Other prerequisites will depend on the story.

### Hints

[ternary][ternary] documentation provides an introduction to ternary expressions.
[throw expressions][throw-expressions] documentation provides details of `throw` expressions.

### After

[ternary][ternary] documentation provides an introduction to ternary expressions.
[throw expressions][throw-expressions] documentation provides details of `throw` expressions.

## Representer

This exercise does not require any specific representation logic to be added to the [representer][representer].

## Analyzer

This exercise does not require any specific analyzer logic to be added to the [analyzer][analyzer].

## Implementing

To implement this exercise, please [follow these instructions](https://github.com/exercism/v3/blob/master/languages/csharp/reference/implementing-a-concept-exercise.md).

## Help

If you have any questions while implementing the exercise, please post the questions as comments in this issue.

[ternary]: https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/conditional-operator
[throw-expressions]: https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-7.0/throw-expression
[analyzer]: https://github.com/exercism/csharp-analyzer
[representer]: https://github.com/exercism/csharp-representer

**Comment by mikedamay** [comment]
Will combine with #1145 Expression Bodied Members

**Comment by mikedamay** [comment]
closed by PR #1880 Expression-bodied-memebers
